webapp/src/components/post_menu_actions/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -4,7 +4,7 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 
-import {id as pluginId} from '../../../manifest';
+import manifest from '../../../manifest';
```

Comment:
I see sometimes we import the manifest as `'manifest'` and sometimes with the relative path. Probably should just do `'manifest'`